doc/user_guide/configuration/index.rst

```diff
@@ -0,0 +1,11 @@
+.. _configuration:
+
+Configuration
+=============================
+
+.. toctree::
+    :maxdepth: 2
+
+    local_configuration
```

Comment:
The overall TOC on top level still looks garbled but maybe you planned to address this in a separate PR?